### PR TITLE
Migrate language from ils,ews. Replaced unused view.

### PR DESCRIPTION
**Pull request by kkrampa**
Depends on: https://github.com/dimagi/logistics/pull/187

### CODE REVIEW

**Comment by dimagimon** [comment]
:+1: No lint errors found.

File: custom/logistics/api.py [hunk 1]
```diff
@@ -212,3 +215,13 @@ def sms_user_sync(self, ilsgateway_smsuser, username_part=None, password=None,
...
                 user.save()
         return user
+
+    def add_language_to_user(self, logistics_sms_user):
+        domain_part = "%s.commcarehq.org" % self.domain
+        username_part = "%s%d" % (logistics_sms_user.name.strip().replace(' ', '.').lower(),
+                                  logistics_sms_user.id)
+        username = "%s@%s" % (username_part[:(128 - (len(domain_part) + 1))], domain_part)
+        user = CouchUser.get_by_username(username)
+        if user:
```
Comment (czue) [inline custom/logistics/api.py]: can you change this to `if user and user.language != logistics_sms_user.language` to avoid unnecessary saves?

**Comment by dimagimon** [comment]
:+1: No lint errors found.

File: custom/ilsgateway/urls.py [hunk 2]
```diff
@@ -14,12 +14,11 @@
...
     url(r'^delete_runs/$', 'delete_reports_runs', name='delete_runs'),
     url(r'^supervision/$', SupervisionDocumentListView.as_view(), name=SupervisionDocumentListView.urlname),
     url(r'^delete_supervision_document/(?P<document_id>\d+)/$', SupervisionDocumentDeleteView.as_view(),
         name='delete_supervision_document'),
-    url(r'^supervision/(?P<document_id>\d+)/$', SupervisionDocumentView.as_view(),
-        name='supervision_document'),
-
+    url(r'^supervision/(?P<document_id>\d+)/$', SupervisionDocumentView.as_view(), name='supervision_document')
 )
+
```
Comment (dimagimon) [inline custom/ilsgateway/urls.py]: W391 blank line at end of file

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by kkrampa** [comment]
@czue Updated

**Pull request merged**